src/components/AppsMenu/helper.js

```diff
@@ -0,0 +1,10 @@
+export const buildEntrypoints = apps => {
+  return apps.flatMap(app => {
+    if (!app.entrypoints) return []
+
+    return app.entrypoints.map(entrypoint => ({
+      ...entrypoint,
+      slug: app.slug
+    }))
+  })
+}
```

Comment:
I hesitated to centralize this code on cozy-client, but as entrypoints are not really mature (specification already changed between yesterday and today), I think it is too soon for this kind of centralization.